epoxy-processor/src/main/java/com/airbnb/epoxy/PoetExtensions.kt

```diff
@@ -102,6 +103,11 @@ private fun JavaClassName.getSimpleNamesInKotlin(): List<String> {
     return originalNames
 }
 
+fun JavaAnnotationSpec.toKPoet(): KotlinAnnotationSpec {
+    val annotationClass = KotlinClassName.bestGuess(type.toString())
```

Comment:
This isn't quite right because it won't capture parameter values in the annotation - so it will only work for basic annotations.

If you're interested it would be great to add that full functionality, otherwise can you add a note warning of this partial implementation?

Reply:
@shaishavgandhi05 great! yes, I think adding tests to `epoxy-processor` makes sense

Reply:
After some investigation, this seems like something really hard to do. Parameters are added as `CodeBlock`'s to the `AnnotationSpec`. There's no way to convert a `JavaCodeBlock` to `KotlinCodeBlock` since `CodeBlock`'s have no getters to know what values they're going to emit.